README.md

```diff
@@ -1,79 +1,55 @@
 # kind-local
 
-This is a Proof of Concept of how to run a local container registry in [Kind](https://github.com/kubernetes-sigs/kind) (Kubernetes in Docker).
+This repo documents the best way to run a local [Kind](https://github.com/kubernetes-sigs/kind) cluster with Tilt.
 
 ## Why
 
 When developing locally, you want to push images to the cluster as fast as possible.
 
-Pushing to an in-cluster container registry skips a lot of overhead:
+Pushing to an in-cluster image registry skips a lot of overhead:
 
 - Unlike with a remote registry, the image stays local to your machine, with no network traffic
 
 - Unlike with `kind load`, docker will skip pushing any layers that already exist in the registry
```

Comment:
```suggestion
- Unlike with `kind load`, you get the benefit of caching (i.e. Docker will skip pushing any layers that already exist in the registry)
```

Reply:
this isn't quite accurate, kind load has some ability to cache